ui/app/templates/components/database-role-setting-form.hbs

```diff
@@ -12,6 +12,16 @@
           <ReadonlyFormField @attr={{attr}} @value={{get @model attr.name}} />
         {{else}}
           <FormField data-test-field={{true}} @attr={{attr}} @model={{@model}} @modelValidations={{@modelValidations}} />
+          {{#if
+            (and
+              (and (eq attr.name "skip_import_rotation") (eq @mode "create") (eq (get @model attr.name) (not @dbSkipImport)))
+            )
+          }}
```

Comment:
when conditionals get this complex it's best to move it to a getter in the component so it's easier to parse. It also doesn't seem like the initial `and` is necessary here, but it's hard to tell.